pkg/tracing/setup.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tracing
+
+import (
+	"fmt"
+
+	"github.com/knative/pkg/configmap"
+	"github.com/knative/pkg/tracing"
+	tracingconfig "github.com/knative/pkg/tracing/config"
+	"github.com/openzipkin/zipkin-go"
+	"go.uber.org/zap"
+)
+
+// TODO Move this to knative/pkg.
```

Comment:
@mdemirhan I'm intending to move this up to knative/pkg once I prove it is useful in eventing. Would you mind reviewing this file?